### PR TITLE
electronmail: Add version 5.1.3

### LINKED ISSUE #10437: [Request]: ElectronMail
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

ElectronMail

### Description

An Electron-based unofficial desktop client for ProtonMail.

### Homepage

https://github.com/vladimiry/ElectronMail

### Download Link(s)

https://github.com/vladimiry/ElectronMail/releases

### Some Indication of Popularity/Repute

**1.3k stars and 90 forks**

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #10437

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/electron-mail.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "5.1.3",
+    "homepage": "https://github.com/vladimiry/ElectronMail",
+    "description": "An Electron-based unofficial desktop client for ProtonMail.",
+    "license": "GPL-3.0",
```
Comment (sitiom) [inline bucket/electron-mail.json:5]:
```suggestion
    "license": "GPL-3.0-only",
```

License can only be `GPL-3.0-only` or `GPL-3.0-or-later`, matching the SPDX identifier.

**Review by sitiom — CHANGES_REQUESTED** [review]

**Review by sitiom — CHANGES_REQUESTED** [review]
Rename manifest to `electronmail`

**Comment by Zliced13** [comment]
Done.

**Review by sitiom — APPROVED** [review]

**Comment by anderlli0053** [comment]
Thanks!

**Pull request merged**